src/api/Feed.js

```diff
@@ -113,6 +113,23 @@ class Feed extends Base {
         this.taskLinksAPI = [];
     }
 
+    addFeedItem(id: string, item: FeedItem) {
+        if (!id || !item) {
+            return;
+        }
+
+        const cachedItems = this.getCachedItems(id);
+        const feedItems = cachedItems ? cachedItems.items : [];
```

Comment:
This can be simplified quite a bit:

```
const { items: feedItems = [] } = this.getCachedItems(id);
```

Reply:
I am going to remove this as discussed and bring it back in when we start hooking into the events